### PR TITLE
new ci stage and deployment

### PR DESCRIPTION
**Pull request by lordfolken**

### CODE REVIEW

File: scripts/check-waypoints [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/python
+
+import sys, getopt
+import aerofiles.seeyou
+
+from aerofiles.seeyou.reader import (
+            Reader as OpenAirReader
+            )
+
+cupfile = OpenAirReader()
```
Comment (Turbo87) [inline scripts/check-waypoints]: SeeYou files are not OpenAir files

File: scripts/check-waypoints [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+print sys.argv[1]
```
Comment (Turbo87) [inline scripts/check-waypoints]: why are you printing the CLI argument?

File: .travis.yml [hunk 3]
```diff
@@ -1,16 +1,12 @@
...
 - cat $TRAVIS_BUILD_DIR/waypoints/*.cup | sort -b > $TRAVIS_BUILD_DIR/waypoints/xcsoar_waypoints.cup
+script:
+- for each in $TRAVIS_BUILD_DIR/waypoints/*.cup ; do $TRAVIS_BUILD_DIR/checks/check-waypoints "${each}" ; done 
 deploy:
   provider: script
   skip_cleanup: true
-  script: ssh-keyscan -p ${DEPLOY_PORT} ${DEPLOY_HOST} > ~/.ssh/known_hosts && rsync -av -e 'ssh -p ${DEPLOY_PORT} -i xcsoar-data-repository_rsa' $TRAVIS_BUILD_DIR/waypoints/ ${DEPLOY_USER}@${DEPLOY_HOST}:${DEPLOY_PATH}/waypoints/ --delete && rsync -av -e 'ssh -p ${DEPLOY_PORT} -i xcsoar-data-repository_rsa' $TRAVIS_BUILD_DIR/airspaces/ ${DEPLOY_USER}@${DEPLOY_HOST}:${DEPLOY_PATH}/airspaces/ --delete
-  on:
+  script: $TRAVIS_BUILD_DIR/scripts/deploy.sh
+ on:
```
Comment (Turbo87) [inline .travis.yml]: indentation seems off here

File: scripts/check-airspaces [hunk 4]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/python
```
Comment (Turbo87) [inline scripts/check-airspaces]: is this script being used anywhere yet?

Comment (lordfolken) [reply scripts/check-airspaces]: Not yet. It should be used in the ci chain to check the airspace files for validity.

File: .travis.yml (hunk 3 above)
Comment (lordfolken) [reply .travis.yml]: http://blog.tgrrtt.com/exploring-the-travisci-configuration-file states the on in the same indent as the script parameter.

**Pull request merged**